karma.conf.sauce.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env node */
+const { createDefaultConfig } = require('@open-wc/testing-karma');
+const merge = require('deepmerge');
+
+const defaultPattern = '+(components|helpers|mixins|templates)/**/*.test.js';
+const customLaunchers = {
+	chrome: {
+		base: 'SauceLabs',
+		browserName: 'chrome',
+		platform: 'OS X 10.13',
+	},
+	firefox: {
+		base: 'SauceLabs',
+		browserName: 'firefox',
+		platform: 'OS X 10.13'
+	},
+	safari: {
+		base: 'SauceLabs',
+		browserName: 'safari',
+		platform: 'OS X 10.13'
+	}
+};
+
+module.exports = config => {
+	config.set(
+		merge(createDefaultConfig(config), {
```

Comment:
Is it possible to use `Object.assign()`?  I think nodejs has supported it for a while.

Reply:
I can give it a try. This is just following [their recommendation](https://open-wc.org/testing/testing-karma.html#manual).

Reply:
Cool... yeah I think it would be nice to avoid if possible.

Reply:
I don't think they're equivalent... deepmerge does a nested merge vs. `Object.assing()` only goes one level deep.

Reply:
Ah 😞 .  I think those open-wc guys are pretty sharp, I was worried about that.